examples/classifier_example.py

```diff
@@ -13,3 +16,16 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+""" Decision tree classifier
+"""
+
+def main():
+    """ Main function.
+    """
+    review_classifier = ReviewClassifier('dt', 'stopwords.txt')
+    review_classifier.train('citalopram_train.csv')
+    review_classifier.classify('neutral.txt')
+
+if __name__ == "__main__":
```

Comment:
This is a special function that Python has that will only run once when the script starts. You only want one of these in the code so it's probably not even reading this one. Best fix is to just remove it.